deepspeed/runtime/zero/parameter_offload.py

```diff
@@ -49,14 +49,18 @@ def __init__(self, parent_module, *args, **kwargs):
         self._parent_module = parent_module
         self._in_forward = False
 
+    def __reduce__(self):
+        r0, _, *r2 = super().__reduce__()
+        return r0, (self._parent_module, ), *r2
```

Comment:
@oraluben - it looks like this syntax isn't supported on python 3.7, is this something you could try to generalize for that as well?

Reply:
That's a surprise 😂 working on it